lib/paypal-adaptive.js

```diff
@@ -241,12 +249,4 @@ adaptivePaymentsMethods.forEach(function (method) {
     };
 });
 
-adaptiveAccountsMethods.forEach(function (method) {
-    var prototypeMethodName = method.charAt(0).toLowerCase() + method.slice(1);
-    var apiMethodName = 'AdaptiveAccounts/' + method;
-    Paypal.prototype[prototypeMethodName] = function (data, callback) {
-        this.callApi(apiMethodName, data, callback);
-    };
-});
-
```

Comment:
Okay, I see. Just noticed that. My bad. I'll make sure to review more carefully before sending a pull request.